src/metabase/models/permissions.clj

```diff
@@ -1322,7 +1322,7 @@
     (throw (ee-permissions-exception perm-type))))
 
 (mu/defn ^:private update-group-permissions!
-  [group-id :- pos-int? new-group-perms :- api.permission-graph/StrictDbGraph]
+  [group-id :- pos-int? new-group-perms :- [:maybe api.permission-graph/StrictDbGraph]]
```

Comment:
Since the DB Graph shapes can be nil over the API, we need to accept nil DB Graphs here.